src/app/wallets/add-invoice-for-wallet.ts

```diff
@@ -97,7 +107,10 @@ export const addInvoiceNoAmountForRecipient = async ({
   const recipientWalletIdChecked = checkedToWalletId(recipientWalletId)
   if (recipientWalletIdChecked instanceof Error) return recipientWalletIdChecked
 
-  const limitOk = await checkRecipientWalletIdRateLimits(recipientWalletIdChecked)
+  const wallet = await getWallet(recipientWalletIdChecked)
```

Comment:
Instead of calling another use case can you just call the repository directly:
```
  const wallets = WalletsRepository().findById(walletId)
```
No need for the indirection